transmissions/lock.py

```diff
@@ -11,7 +11,7 @@ def lock(key, timeout=5000):
     """
 
     lock_id = 'lock-transmission-{0}'.format(key)
-    acquire_lock = lambda: cache.add(lock_id, 1, timeout)
+    acquire_lock = lambda: cache.add(lock_id, 1, 90) # fix to keep the key for 90secs in redis instead of 5000sec
```

Comment:
Feels like updating the default value of `timeout` would have been a better approach

Reply:
The acquire try timeout and the cache.add timeout are two different things. The former is how long you want to wait until you give up, the latter is how long the lock will be "valid" for (=TTL of the redis entry). So the "proper" fix would've been to add a new variable and use it here, but we discussed that locking overall needs consolidation in the near future and did the quickest fix.

Reply:
As long as you're on top of it 👍I agree the 2 variables make sense to split the 2 meanings that used to be one and the same.